website/docs/intro.md

```diff
@@ -3,30 +3,34 @@ sidebar_position: 1
 title: "Contributor Guide"
 ---
 
-
 ## We ♥️ Contributions!
-
 `awesome-azd` is a community-friendly resource to help developer discover, create, and share, _azd-template_ resources for streamlining your application development and CI/CD pipelines. It is an _open-source_ effort open for contributions in many ways. 
 
-
 ## Our Custom Issues
-
 The [custom issues](https://github.com/Azure/awesome-azd/issues/new/choose) page shows the different ways in which _anyone_ can contribute to make this project successful for _everyone_. 
-Here are the 4 main ones:
 
- - submit a **resource** for the `awesome-azd/README.md` list
- - submit an **idea** for a template
- - contribute a **requested template** matching an idea
- - contribute a **new template** from your own project or idea
+### Submit a Template Idea
+There are ample ways to combine languages with Azure services and tools to create applications, so if you cannot find a template with architecture that works for you-- you can submit a request for that template. 
+
+Keep in mind, a template can be used far beyond it's original purpose. For example, a [ToDo Application in C# with Azure SQL and App Service](https://github.com/Azure-Samples/todo-csharp-sql) can be modified and used to create a grocery list application. 
```

Comment:
For clarity, maybe we can say enforce that a template is intended to be extensible. The idea here is that you find a template that seems like it uses a simple architecture and then you can customize it (e.g. swap out the source code).